### PR TITLE
Moving the sensor app to appscale repo.

### PR DESCRIPTION
**Pull request by tmarballi**
https://ocd.appscale.com:8080/job/Daily%20Build/1296/

### CODE REVIEW

File: Apps/sensor/custom_exceptions.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+""" Custom Backup and Recovery exceptions. """
+
+class BackupValidationException(Exception):
```
Comment (menivaitsi) [inline Apps/sensor/custom_exceptions.py]: Are there used anywhere? If not, please remove.

Comment (tmarballi) [inline Apps/sensor/custom_exceptions.py]: I should remove it, it was used by another utils class which I got rid of.

File: Apps/sensor/common/constants.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  TEST_MODE = True
+  DEBUG_MODE = True
+
+# High level tags use to communicate between the server and client.
+class ApiTags(object):
+  """ A class containing shared constants. """
+  ALL_TESTS = "all"
+  API_KEY = "api_key"
+  APP_ID = "app_id"
+  CREDS = "creds"
```
Comment (menivaitsi) [inline Apps/sensor/common/constants.py]: Not used, please remove.

File: Apps/sensor/common/constants.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  DATA = "data"
+  DISPLAY = "display"
```
Comment (menivaitsi) [inline Apps/sensor/common/constants.py]: Same here.

File: Apps/sensor/common/constants.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+  ERROR = "error"
+  STATUS = "status"
+  REFRESH_TOKEN = "refresh_token"
```
Comment (menivaitsi) [inline Apps/sensor/common/constants.py]: Same here

File: Apps/sensor/common/constants.py [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  REQUEST_TOKEN = "request_token"
```
Comment (menivaitsi) [inline Apps/sensor/common/constants.py]: Same here.

File: Apps/sensor/common/constants.py [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  SECRET_KEY = "shared_key"
```
Comment (menivaitsi) [inline Apps/sensor/common/constants.py]: Same here.

File: Apps/sensor/common/tools/gruntfile.js [hunk 7]
```diff
@@ -0,0 +1,67 @@
+module.exports = function(grunt) {
```
Comment (menivaitsi) [inline Apps/sensor/common/tools/gruntfile.js]: Remove this file.

File: Apps/sensor/common/tools/package.json [hunk 8]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (menivaitsi) [inline Apps/sensor/common/tools/package.json]: Remove this file.

File: Apps/sensor/common/util.py [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+def is_gae():
+  """ Determine if we are running on GAE.
+
+  Returns:
+    True if running on GAE, False otherwise.
+  """
+  return 'SERVER_SOFTWARE' in os.environ and \
+    os.environ['SERVER_SOFTWARE'].startswith('Google')
+
+def timesince(dt, default="just now"):
```
Comment (menivaitsi) [inline Apps/sensor/common/util.py]: Function not used.

File: Apps/sensor/common/util.py [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+def is_appscale():
+  """ Determine if we are running on AppScale.
+
+  Returns:
+    True if on AppScale, False otherwise.
+  """
+  return 'SERVER_SOFTWARE' in os.environ and \
+    os.environ['SERVER_SOFTWARE'].startswith('AppScale')
+
+def is_gae():
```
Comment (menivaitsi) [inline Apps/sensor/common/util.py]: Function not used.

File: Apps/sensor/common/util.py [hunk 11]
```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
```
Comment (menivaitsi) [inline Apps/sensor/common/util.py]: Remove unused functions from this file.

File: Apps/sensor/common/constants.py [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+MAX_STRING_LENGTH = 10000
+
+# Number of samples to test an API.
+NUM_SAMPLES = 1
+
+# Convert multiplier from seconds to milliseconds.
+SECONDS_TO_MILLI = 1000
+
+# Whether or not we're in test mode.
+TEST_MODE = False
```
Comment (menivaitsi) [inline Apps/sensor/common/constants.py]: This is not used, please remove.

File: Apps/sensor/LICENSE [hunk 13]
```diff
@@ -0,0 +1,2 @@
+Copyright (c) 2014 AppScale Systems, Inc.
+This is proprietary software. All rights reserved. No warrenty is provided.
```
Comment (obino) [inline Apps/sensor/LICENSE]: This will be added with the same Apache license as appscale.

**Comment by tmarballi** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/

**Pull request merged**